### PR TITLE
Add Animal Crossing: Happy Home Designer

### PR DESCRIPTION
**Pull request by Arianami**
Adds Animal Crossing: Happy Home Designer.

*Automatically generated by PR builder*

### CODE REVIEW

File: animal-crossing-happy-home-designer/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Animal Crossing: Happy Home Designer"
+description = "Animal Crossing: Happy Home Designer is a sandbox video game developed and published by Nintendo, with assistance from Monolith Soft, for the Nintendo 3DS. The game was released in Japan in July 2015, in North America in September 2015, and in PAL regions in October 2015. The game is a spin-off of the Animal Crossing series where the player has to design homes for various anthropomorphic animal characters. (Source: https://en.wikipedia.org/wiki/Animal_Crossing:_Happy_Home_Designer )"
```
Comment (Leo626) [inline animal-crossing-happy-home-designer/game.dat]: You don't need to have the release dates and source in the description. It should be kept short.

File: animal-crossing-happy-home-designer/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+title = "000400000014F200"
+region = "eur"
+release_date = "2015-07-30"
+
+[[testcases]]
+title = "000400000014F200"
+compatibility = "4"
+date = "2017-07-13"
+version = "HEAD-06d4564"
+cpu = "Intel Xeon"
```
Comment (TheKoopaKingdom) [inline animal-crossing-happy-home-designer/game.dat]: According to [this](https://www.intel.com/content/www/us/en/products/processors/xeon.html), there's multiple different CPUs in the Xeon family, so you should be more specific.

**Comment by Schplee** [comment]
@Arianami Please fix the remaining issues, otherwise we will close the PR.

**Comment by Schplee** [comment]
No sign of PR author - closing.

**Pull request closed without merging**